tests/unit/helpers.js

```diff
@@ -404,6 +404,8 @@ export function createUserProfileResponse({
   };
 }
 
+export const createUserAccountResponse = createUserProfileResponse;
```

Comment:
In the future this might be different but for now I just exported the reference so that if we changed it in the future we can just do it at the source.